pkg/ratelimit/limiter.go

```diff
@@ -20,8 +20,19 @@ import (
 	"golang.org/x/time/rate"
 )
 
+// DimensionConfig is the limit dimension config of one label
+type DimensionConfig struct {
+	// qps conifg
+	QPS      float64
+	QPSBurst int
+	// concurrency config
+	ConcurrencyLimit uint64
+}
+
 // Limiter is a controller for the request rate.
 type Limiter struct {
+	configMux          sync.Mutex
+	labelConfig        map[string]DimensionConfig
```

Comment:
Could we use pointer instead ?
```suggestion
	labelConfig        map[string]*DimensionConfig
```